features/st_private_deposit.feature

```diff
@@ -0,0 +1,23 @@
+Feature: Conduct a special transaction - private deposit behavior
+
+  Background:
+
+         When I'm logged in with 1234
+
+  Scenario: As a valid user I conduct a private deposit and I validate the transaction in the Bonjournals transaction details
+
+
+         Then I press "LocafoxMenuBtn"
+         Then I press "Einlage/Ausgabe"
+         Then I scroll "SpecialTransactionsTypeSpinner" to text "Einlage" and touch it
+         Then I press "SpecialTransactionsNoteEditText"
+         Then I enter text "Einlage" into field with id "SpecialTransactionsNoteEditText"
+         Then I hide keyboard
+         Then I press 2000
```

Comment:
What does this do if the keyboard is hidden?

Reply:
The keyboard is blocking the Pinpad and I had to implement this extra step for the feature not to fail.